File: tools/project-memory-context/tests/fixtures/sample.cs
namespace Example;

public interface IGreetable
{
    string Greet();
}

public class Animal : IGreetable
{
    public string Greet() => "Hello";
    protected void Breathe() {}
    private void Sleep() {}
}

public enum Status
{
    Active,
    Inactive
}

public record PersonRecord(string Name, int Age);

File: tools/project-memory-context/tests/fixtures/sample.java
package com.example;

public interface Greetable {
    String greet();
}

public class Animal {
    public void speak() {}
    protected void breathe() {}
    private void sleep() {}
}

public enum Status {
    ACTIVE, INACTIVE
}

abstract class Base {
    public abstract void run();
}
